cltk/__init__.py

```diff
@@ -0,0 +1,72 @@
+# Classical Language Toolkit (NLTK)
+#
+# Copyright (c) 2016 Kyle P. Johnson.
+# Authors: Kyle P. Johnson.
+# URL: <http://cltk.org/>
+# For license information, see LICENSE
+
+"""
+The Classical Language Toolkit (CLTK) offers natural language
+processing support for Classical languages. CLTK is written
+entirely in python.
+
+The goals of the Classical Language Toolkit (CLTK) are to:
+
+    compile analysis-friendly corpora in a variety of Classical
+    languages
+
+    gather, improve, and generate linguistic data required for NLP
+
+    develop a free and open platform for generating reproducible,
+    scientific research that advances the study of the languages
+    and literatures of the ancient world.
+
+
+
+See the webpage for more information and documentation:
+    http://cltk.org
+"""
+from pkg_resources import get_distribution
+
+# //////////////////////////////////////////////////////
+# Metadata
+# //////////////////////////////////////////////////////
+
+# Version.  For each new release, the version number should be updated
+# in the file VERSION.
+
+import sys
+if sys.version_info[0] != 3:
+    raise ImportError("Python Version 3 or above is required for cltk.")
```

Comment:
a bit nitpicky, but you should lowercase `Version`. I personally think its fine to just check for any version of python3, but there's also argument for specifically checking 3.4 / 3.5 since we can ensure that builds are error-free on them (since travis runs those two). It might be a bit restrictive though. Regardless, I'm probably not qualified to pass judgement on this.